inference-engine/src/transformations/include/transformations/rt_info/fused_names_attribute.hpp

```diff
@@ -86,13 +91,23 @@ extern template class TRANSFORMATIONS_API VariantImpl<ngraph::FusedNames>;
 template<>
 class TRANSFORMATIONS_API VariantWrapper<ngraph::FusedNames> : public VariantImpl<ngraph::FusedNames> {
 public:
-    OPENVINO_RTTI("Variant::RuntimeAttribute::FusedNames");
-    BWDCMP_RTTI_DECLARATION;
+    OPENVINO_RTTI("fused_names", "0");
```

Comment:
Can we use `VariantWrapper<ngraph::FusedNames>` instead?

Reply:
no. this name will be used for serialization so IR will look like hell with this huge not useful names.